css/style.css

```diff
@@ -115,3 +135,282 @@ header {
+}
+
+.work-title {
+  position: static;
+  width: 496px;
+  height: 46px;
+  left: 0;
+  top: 0;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  color: #091e42;
+  float: left;
+  padding: 30px;
+}
+
+.work-info ul li {
+  display: inline;
+  list-style: circle;
+  padding-bottom: 30px;
+  padding-right: 7px;
+  color: #c1c7d0;
+  font-size: 25px;
+}
+
+.work-tech {
+  padding-top: 60px;
+}
+
+.work-tech ul li {
+  display: inline;
+  align-items: center;
+  background: #ebebff;
+  border-radius: 8px;
+  width: 49px;
+  height: 20px;
+  font-size: 21px;
+  padding: 4px 12px;
+  color: #6070ff;
+}
+
+.work-info {
+  padding-top: 30px;
+}
+
+.intro {
+  text-transform: uppercase;
+  color: #000;
+}
+
+.work-info i {
+  color: #c1c7d0;
+  font-size: 18px;
+}
+
+.description {
+  padding: 40px;
+  position: static;
+  width: 496px;
+  height: 50px;
+  left: 24px;
+  top: 146px;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 24px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 24px 0;
+  word-break: break-all;
+  word-wrap: break-word;
+}
+
+.btn-one {
+  color: #396df2;
+  border: 1px solid #396df2;
+  display: flex;
+  align-content: center;
+  margin: 35px;
+  padding: 10px;
+  border-radius: 5%;
+  font-size: 24px;
+}
+
+.btn-two {
+  color: #fff;
+  background: #4053fc;
+  border: 1px solid #396df2;
+  display: flex;
+  align-content: center;
+  margin: 35px;
+  padding: 10px;
+  border-radius: 5%;
+  font-size: 24px;
+}
+
+.about-myself {
+  padding-top: 100px;
+  padding-bottom: 100px;
+  border-top-right-radius: 110px;
+  background-image: url("../image/header_background-v.png");
+}
+
+.about-myself .container {
+  background: transparent;
+}
+
+.section-contact {
+  padding-top: 100px;
+  padding-bottom: 100px;
+  border-top-left-radius: 110px;
+  background: white;
+  align-content: center;
+  text-align: center;
+  background-image: url("../image/header_background-b.png");
+}
+
+.section-contact .container {
+  background: transparent;
+}
+
+.section-contact h1 {
+  color: #fff;
+  font-weight: 700;
+  font-size: 48px;
+  text-align: center;
+}
+
+.section-contact p {
+  font-size: 30px;
+  color: white;
+  text-align: center;
+}
+
+.section-contact input {
+  height: 50px;
+  font-size: 20px;
+  font-weight: 500;
+}
+
+.grid-container {
+  display: grid;
+  align-content: center;
+  grid-template-columns: auto auto auto;
+  grid-gap: 10px;
+  padding: 10px;
+  border: 0;
+  background: transparent;
+}
+
+.grid-container > li img {
+  position: relative;
+  top: -12px;
+  left: -18px;
+}
+
+.grid-container > li p {
+  font-size: 18px;
+  padding-top: 8px;
+}
+
+.stacks {
+  padding-top: 30px;
+}
+
+.section-contact button {
+  display: flex;
+  justify-content: center;
+}
+
+textarea {
+  width: 537px;
+}
+
+@media (max-width: 800px) {
+  .flex-container {
+    flex-direction: column;
+  }
+
+  .work-section .container {
```

Comment:
- [ ] [OPTIONAL] Consider removing all bootstrap classes applied in the HTML file. This project is to test your pure HTML/CSS knowledge. Problems come in when you try to restyle bootstrap classes. For example, Bootstrap already has a `.container ` class which has its own special properties. Applying this class in the HTML and then trying to restyle it as a class independent of Bootsrap, creates conflict and most of the time, Bootstrap overrides your own style, and the page doesn't get rendered as expected.